projects/1.20/assets/veggies-delight/veggiesdelight/lang/zh_cn.json

```
@@ -52,64 +52,64 @@
+  "item.veggiesdelight.dandelion_juice": "奶香蒲公英汁",
   "item.veggiesdelight.dandelion_leaf": "蒲公英叶",
 
-  "block.veggiesdelight.vegan_pizza": "全素披萨",
+  "block.veggiesdelight.vegan_pizza": "纯素披萨",
   "block.veggiesdelight.sweet_potato_pie": "红薯派",
   "block.veggiesdelight.beetroot_brownie_tray": "甜菜根布朗尼",
 
-  "block.veggiesdelight.mature_dandelion": "成熟的蒲公英",
-  "block.veggiesdelight.wild_bellpeppers": "野生甜椒",
-  "block.veggiesdelight.wild_cauliflowers": "野生花椰菜",
+  "block.veggiesdelight.mature_dandelion": "成熟蒲公英",
+  "block.veggiesdelight.wild_bellpeppers": "野生甜椒丛",
+  "block.veggiesdelight.wild_cauliflowers": "野生花椰菜丛",
   "block.veggiesdelight.wild_garlic": "野生大蒜",
   "block.veggiesdelight.wild_sweet_potatoes": "野生红薯",
 
   "creativetab.veggiesdelight_tab": "蔬菜乐事",
 
   "veggiesdelight.advancement.root": "蔬菜乐事",
-  "veggiesdelight.advancement.root.desc": "去采摘你的蔬菜！",
+  "veggiesdelight.advancement.root.desc": "快去获取你的蔬菜吧！",
 
-  "veggiesdelight.advancement.fruit": "这不是水果吗？",
+  "veggiesdelight.advancement.fruit": "这难道不是水果吗？",
   "veggiesdelight.advancement.fruit.desc": "我们中出了一个叛徒",
 
-  "veggiesdelight.advancement.first": "素来如此",
-  "veggiesdelight.advancement.first.desc": "收集你的第一个蔬菜",
+  "veggiesdelight.advancement.first": "生菜起步",
+  "veggiesdelight.advancement.first.desc": "收获你的第一份蔬菜",
 
-  "veggiesdelight.advancement.all": "Gotta catch 'em all",
-  "veggiesdelight.advancement.all.desc": "收集所有蔬菜",
+  "veggiesdelight.advancement.all": "收集全餐",
+  "veggiesdelight.advancement.all.desc": "集齐所有种类的蔬菜",
 
-  "veggiesdelight.advancement.eat": "虽然是生的，但也能吃",
+  "veggiesdelight.advancement.eat": "虽然生但还能吃",
   "veggiesdelight.advancement.eat.desc": "生吃任意一种蔬菜",
 
   "veggiesdelight.advancement.pizza_time": "披萨时间到！",
-  "veggiesdelight.advancement.pizza_time.desc": "制作你的第一个全素披萨",
+  "veggiesdelight.advancement.pizza_time.desc": "制作你的第一份纯素披萨",
 
-  "veggiesdelight.advancement.burger": "汉堡开花",
-  "veggiesdelight.advancement.burger.desc": "制作一个花椰菜汉堡",
+  "veggiesdelight.advancement.burger": "汉堡绽放",
+  "veggiesdelight.advancement.burger.desc": "制作一份花椰菜素汉堡",
 
-  "veggiesdelight.advancement.cauliflower": "素食之道",
-  "veggiesdelight.advancement.cauliflower.desc": "油炸一些花椰菜花球，素食者的炸鸡",
+  "veggiesdelight.advancement.cauliflower": "纯素之道",
+  "veggiesdelight.advancement.cauliflower.desc": "油炸花椰菜——素食版炸鸡",
 
-  "veggiesdelight.advancement.pie": "人生一派",
-  "veggiesdelight.advancement.pie.desc": "制作一个红薯派",
+  "veggiesdelight.advancement.pie": "品味生活",
+  "veggiesdelight.advancement.pie.desc": "烘焙红薯派",
 
   "veggiesdelight.advancement.med": "自然疗法",
-  "veggiesdelight.advancement.med.desc": "制作一份糖蒜",
+  "veggiesdelight.advancement.med.desc": "制作一瓶发酵蒜蜜",
 
   "veggiesdelight.advancement.cesar": "西撒————！",
   "veggiesdelight.advancement.cesar.desc": "制作一份凯撒沙拉",
 
   "veggiesdelight.advancement.bellpepper_master": "胡椒博士",
-  "veggiesdelight.advancement.bellpepper_master.desc": "制作所有需要甜椒的菜",
+  "veggiesdelight.advancement.bellpepper_master.desc": "用甜椒制作所有可能的菜肴",

```

```suggestion
  "veggiesdelight.advancement.bellpepper_master.desc": "制作所有需要甜椒的菜",
```
原翻就ok的
这里every dish possible就是指`所有的菜`，`可能`属于生硬翻译

---

OK